openapi/openapi.yaml

```
@@ -2,6 +2,121 @@ openapi: 3.0.3
+   Nombre_Recambio:
+     title: 'Nombre recambio'
+     type: string
+   Descripcion_Recambio:
+     title: 'Descripcion'
+     description: >
+       Explicación detallada de las características, funciones y especificaciones
+       técnicas de una pieza o componente de repuesto para un vehículo. Esta descripción
+       es importante para asegurarse de que el recambio es adecuado para el vehículo y 
+       para comprender su función dentro del mismo.
+     type: string
+
+   Proveedor_Recambio:
+      title: 'Proveedor del recambio'
+      description: >
+       El proveedor de un recambio para un taller de vehículos puede ser una empresa
+       especializada en la fabricación y distribución de piezas de repuesto para automóviles
+       o bien un distribuidor que suministre los repuestos de diferentes fabricantes. 
+      type: string
+   Equivalencias_Recambio:
+      title: 'Equivalencias'
+      description: >
+        Las equivalencias de un recambio para un vehículo se refieren a piezas o componentes que,
+        aunque no sean exactamente iguales en diseño o especificaciones técnicas a la pieza 
+        original del vehículo, pueden ser utilizadas como reemplazo para dicha pieza.
+      type: string
+      items:
+         type: object
+         properties:
+           nombre: 
+             type: string
+             description: 'Nombre de la pieza equivalente'
+           fabricante:
+             type: string
+             description: 'Nombre del fabricante de la pieza equivalente'
+           modelo:
+             type: string
+             description: 'Modelo de vehículo compatible con la pieza equivalente'
+   Garantia_Recambio:
+      title: 'Garantía del recambio'
+      description: >
+        La garantía de un recambio de un vehículo es un compromiso que el fabricante o vendedor 
+        del recambio adquiere con el comprador, en el que se asegura que el recambio funcionará 
+        de manera adecuada y sin defectos durante un periodo de tiempo determinado.
+      type: string
+        
+   Precio_Recambio:
+      title: 'Precio'
+      description: 'Precio sin IVA asignado a cada recambio'
+      type: number
+      format: double
+   IVA_Recambio:
+      title: 'IVA '
+      description: >
+       Es un tributo de carácter indirecto. Recae sobre los consumidores,
+       puesto que lo pagan al adquirir un producto o servicio: sobre el precio neto 
+       de los mismos se establece una aplicación que varía en su porcentaje.
+      type: string
+      pattern: '/^[0-9]{1,2}[%]$/'
+      example: '21%'
+       # https://v6.fakerjs.dev/guide/
+      x-faker: helpers.regexpStyleStringParse('/^[0-9]{1,2}[%]$/')
+   Importe_Recambio:
+      title: 'Importe'
+      description: 'Precio final añadiendo el IVA al precio a cada recambio'
+      type: number
+      format: double
+   Cantidad_Recambio:
+      title: 'Cantidad'
+      description: 'Numero de unidades del recambio disponibles en el taller'
+      type: integer
```

```suggestion
      type: integer
      minimum: 0
      maximum: 10000
```
Lo del maximo lo puedes cambiar o eliminarlo.